src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -111,9 +111,11 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "attrs": ("attr", "attrs"),
     "auth0-python": ("auth0",),
     "beautifulsoup4": ("bs4",),
+    "biopython": ("Bio", "BioSQL",),
```

Comment:
Like you identify with `setuptools`, I suspect these trailing commas aren't required. It was probably this one that caused the listing failure originally:

```suggestion
    "biopython": ("Bio", "BioSQL"),
```

Plus reverting the others.